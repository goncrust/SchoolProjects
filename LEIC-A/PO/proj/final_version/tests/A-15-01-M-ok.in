4
3
969001
8
969006
0123456789
0
0
5
1
0
4
3
969011
8
969001
0123456789012345678901234567890123456789012345678
0
0
5
1
0
0
