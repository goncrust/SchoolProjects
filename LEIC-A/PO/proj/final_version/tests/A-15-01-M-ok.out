MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: TEXT|1|969001|969006|10|10|FINISHED
CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: TEXT|1|969001|969006|10|10|FINISHED
TEXT|2|969011|969001|49|10|FINISHED
CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 